### ISSUE TITLE
测试的时候满屏标框是什么情况呢

### ISSUE THREAD

**Issue by jun214384468** [zylo117/Yet-Another-EfficientDet-Pytorch#527]
![image](https://user-images.githubusercontent.com/37968425/96553889-61f39c80-12e8-11eb-9d8e-d392ea512fd9.png)
求解释

**Comment by zylo117** [comment]
underfitting

**Comment by jun214384468** [comment]
> underfitting

我比较了10批次与100批次结果都一样呀

**Comment by siyuch-fdu** [comment]
我也是满屏标注。我感觉是训练直接爆了，cls loss有几万多，但是尝试调整了学习率依然没办法把cls loss调到正常值。

**Comment by siyuch-fdu** [comment]
目前正常了，确实是underfitting，readme上面写的lr太小了，我把它从1e-5调到了1e-2，loss马上就降下来了

**Comment by jun214384468** [comment]
> 确实是underfitting，readme上面写的lr太小了，我把它从1e-5调到了1e-2，loss马上就降下来了

我的loss正常啊  但是检测还是有问题

**Comment by zylo117** [comment]
It's just an example, you'll have to tune the lr by yourself.

**Comment by jun214384468** [comment]
> It's just an example, you'll have to tune the lr by yourself.

Thank you very much for taking the time to reply，This is my loss graph，
![image](https://user-images.githubusercontent.com/37968425/97523081-453c2080-19dc-11eb-8a6f-c9a83c46fafc.png)

**Comment by zylo117** [comment]
that's overfitting,after all you set lr to 1e-2

**Comment by jun214384468** [comment]
> that's overfitting,after all you set lr to 1e-2

Ok, thank you very much

**Comment by jun214384468** [comment]
> 目前正常了，确实是underfitting，readme上面写的lr太小了，我把它从1e-5调到了1e-2，loss马上就降下来了

请问你的测试正常了吗？有没有提出一个保存的模型进行测试呢

**Comment by qubapikaqi** [comment]
我把学习率变大之后，用voc数据集训练精度只能到三十多，batch_size设为96，我不知道是什么问题，用的d0预训练模型

**Comment by jun214384468** [comment]
> 我把学习率变大之后，用voc数据集训练精度只能到三十多，batch_size设为96，我不知道是什么问题，用的d0预训练模型

你尝试把batch_size放小点。
在这个模型下voc数据集怎么训练的呢？你是改了数据加载的代码吗？

**Comment by zylo117** [comment]
96可还行，宁是用了多大的卡啊，放那么多图

**Comment by qubapikaqi** [comment]
我转的数据集格式，改数据读取方式确实是有点难为我了

**Comment by qubapikaqi** [comment]
我改小一下batch_size试一下，96确实有点夸张了，我第一次用四张3090没经验，哈哈

**Comment by qubapikaqi** [comment]
请问微调的预训练权重是哪个呀

**Comment by siyuch-fdu** [comment]
> 我改小一下batch_size试一下，96确实有点夸张了，我第一次用四张3090没经验，哈哈

4张3090也太顶了吧。。所以想问下3090用下来有什么坑吗，有没有出现什么问题？

**Comment by qubapikaqi** [comment]
坑倒是没有，就是cuda必须11以上，pytorch要1.7------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;&quot;siyuch-fdu&quot;<notifications@github.com&gt;
发送时间:&nbsp;2020年11月13日(星期五) 下午2:51
收件人:&nbsp;&quot;zylo117/Yet-Another-EfficientDet-Pytorch&quot;<Yet-Another-EfficientDet-Pytorch@noreply.github.com&gt;;
抄送:&nbsp;&quot;qubapikaqi&quot;<1602220151@qq.com&gt;;&quot;Comment&quot;<comment@noreply.github.com&gt;;
主题:&nbsp;Re: [zylo117/Yet-Another-EfficientDet-Pytorch] 测试的时候满屏标框是什么情况呢 (#527)

**Comment by juliusyang97** [comment]
我也遇到了同样的问题
![Uploading image.png…]()
